webauthn/authentication/verify_authentication_response.py

```diff
@@ -42,7 +43,7 @@ class VerifiedAuthentication(WebAuthnBaseModel):
 
 def verify_authentication_response(
     *,
-    credential: AuthenticationCredential,
+    credential: Union[str, bytes, bytearray, AuthenticationCredential],
```

Comment:
I'm curious why you included `bytes` and `bytearray` here. I'm fine with this property gaining the ability to accept raw JSON as `str` which we'll then parse to `AuthenticationCredential`, but I don't immediately see why we'd want to include these other two data types (even if Pydantic has no problem parsing them.)

I have the same question for `verify_registration_response()` too but I'll won't repost this down there.

Reply:
I just didn't want to assume stricter typing was OK but I also don't see uses besides `str`. Fixed.